src/info.json

```diff
@@ -1,10 +1,13 @@
 {
-    "name": "clusterio",
-    "version": "1.17.1",
-    "title": "Clusterio",
-    "author": "Danielv123 & Keyboardhack & justarandomgeek & AreYouScared & psihius & Godmave",
-    "contact": "danielv@live.no",
+    "name": "subspace_storage",
+    "version": "2.0.0",
+    "title": "Subspace Storage",
+    "author": "Clusterio Team",
     "homepage": "https://github.com/clusterio/factorioClusterio",
-    "description": "Allows transfer of items, fluids, research and signals between servers. *Server owners, see https://github.com/clusterio/factorioClusterio for installation instructions*",
-    "factorio_version": "0.18"
+    "description": "Allows storing items and fluids in a limitless shared subspace.",
+    "factorio_version": "0.17",
```

Comment:
Is this backwards compatible with 0.17? The mod already was on 0.18+

Reply:
I target stable first and foremost and the icon sizes was the only thing that changed from 0.17 to 0.18 which I reverted here.  Once this is usable I will look into supporting both 0.17 and 0.18.